.github/workflows/runbook-freshness-check.yaml

```diff
@@ -1,49 +1,81 @@
 name: Runbook Freshness Check
 
+# Security: Limit permissions to minimum required
 permissions:
-  contents: read       # checkout + git log
-  issues: write        # create / update issues
-  pull-requests: write # allow comments if needed in the future
+  contents: read       # Required: checkout repo + git log for file history
+  issues: write        # Required: create and update issues for stale runbooks
+  # Explicitly deny all other permissions
+  actions: none
+  checks: none
+  deployments: none
+  id-token: none
+  packages: none
+  pages: none
+  pull-requests: none
+  repository-projects: none
+  security-events: none
+  statuses: none
 
 on:
   schedule:
     # Run daily at 9 AM UTC
     - cron: '0 9 * * *'
```

Comment:
If we care about this being fresh every 90 days 

Why are we running every single day? 

Why not `0 0 1 */3 *` (every 3 months) or `0 0 1 */1 *` (every month) or even once on a sunday `0 9 * * sun`

Reply:
no good reason, I'll change it